3 - PB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -1757,6 +1757,82 @@ Questo sprint impiega:
   supplement: "Grafico"
 )
 
+== Sprint 17 dal 26-02-2024 al 03-03-2024
+
+=== Obiettivi prefissati
+In questo sprint si comincia il lavoro di definizione del design e si prosegue il lavoro sulla documentazione. Gli obiettivi prefissati sono:
+- meeting interno di design thinking, in modalità sincrona, il giorno 26/02/2024 ore 15:00;
+- colloquio con il Proponente, in modalità sincrona, il giorno 29/02/2024 alle ore 16:00;
+- estendere le #ndp con il paragrafo di Technical processes/Implementation process;
+- estendere le #ndp con il paragrafo di Technical processes/Integration process;
+- estendere le #ndp con il paragrafo di Technical processes/Verification process;
+- estendere le #ndp con il paragrafo di Organizational Project-Enabling processes/Quality Management process;
+- aggiornare #ndp normando il sistema di tracciamento rischi - risoluzione;
+- revisione del documento #adr;
+- redigere introduzione del documento Manuale Utente;
+- redigere introduzione del documento Specifica Tecnica;
+- redigere preventivo e consuntivo dello sprint 16 nel #pdp;
```

Comment:
i punti dell'elenco che finiscono con un #documento poi compilandoli non mi vede il ;
fammi sapere se anche a te fa cosi, non sarebbe colpa tua ma forse è da fare presente o trovare una soluzione alternativa.

Reply:
ok, come ho detto ieri in riunione non riesco a compilarlo questo documento quindi non potevo saperlo. vedo di aggirare la cosa